shell/apps/web/index.debug.html

```diff
@@ -7,12 +7,18 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 
```

Comment:
This (index.debug.html) is deprecated or should be removed?

Reply:
I figured we should leave it deprecated for a while until the word gets out about the URL parameter.